<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1854</width>
    <height>565</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>肌电数据同步采集显示</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="lineEdit_IP">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_datasaving2">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>340</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>保存数据</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Connect">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>连 接</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Send">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>450</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>发 送</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_Recv">
    <property name="geometry">
     <rect>
      <x>379</x>
      <y>10</y>
      <width>561</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_Send">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>450</y>
      <width>421</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>IP地址：</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_Recv_data">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>351</width>
      <height>111</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">
border-image: url(:/myx/1.jpg);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>端  口：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Port">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_dataclear_2">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>340</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>清除数据</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>10</y>
      <width>771</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout"/>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>250</y>
      <width>771</width>
      <height>231</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2"/>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>1740</x>
      <y>40</y>
      <width>105</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QRadioButton" name="emg1">
       <property name="text">
        <string>肌电通道1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="emg2">
       <property name="text">
        <string>肌电通道2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="emg3">
       <property name="text">
        <string>肌电通道3</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="emg4">
       <property name="text">
        <string>肌电通道4</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>1740</x>
      <y>300</y>
      <width>101</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QRadioButton" name="emg5">
       <property name="text">
        <string>肌电通道5</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="emg6">
       <property name="text">
        <string>肌电通道6</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="emg7">
       <property name="text">
        <string>肌电通道7</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="emg8">
       <property name="text">
        <string>肌电通道8</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>330</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>数据行数：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>330</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>380</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>输出到文件夹：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>381</x>
      <y>410</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>输出文件名称：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="outputname">
    <property name="geometry">
     <rect>
      <x>491</x>
      <y>410</y>
      <width>401</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="outputloc">
    <property name="geometry">
     <rect>
      <x>491</x>
      <y>380</y>
      <width>401</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_outputloc">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>380</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>…</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>361</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
border-image: url(:/myx/emg.png);</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1854</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
